.github/workflows/release-workflow.yaml

```diff
@@ -1,34 +1,28 @@
 name: Release new version
 
 on:
-  workflow_call:
+  workflow_dispatch:
     inputs:
       majorRelease:
         required: false
-        type: string
         description: 'Release as version. Example values are: minor, major, 5.6.1, 1.0.0. Leave empty for regular release.'
-      branch:
-        required: true
-        type: string
-        description: 'main or 0.x branch'
       tag:
         required: false
-        type: string
         description: 'Prerelease tag name. Leave empty for regular release.'
 
 jobs:
-  # integration-tests:
-  #   uses: ./.github/workflows/integration-tests.yaml
```

Comment:
You'll also need to change `integration-tests.yaml` to call v1 and v2 counterparts.